docs/assembly.rst

```diff
@@ -699,7 +694,7 @@ The following assembly will be generated::
         mstore(ret, r)
         return(ret, 0x20)
       }
-      default: { jump(invalidJumpLabel) }
+      default: { revert(0, 0) }
```

Comment:
Actually, currently it inserts the invalid opcode if no fallback is specified, so we should use that here.